### DISCUSSION TITLE
Image storeOriginalName not working with s3

### DISCUSSION THREAD

**Discussion by gepopp** [laravel/nova-issues#4121] [Q&A] [answered]
- Laravel Version: 9.1
- Nova Version: 4.3
- PHP Version: 8.1

### Description:

if i swith an Image field from:

```
use Laravel\Nova\Fields\Image as ImageField;

....

  ImageField::make( 'Bild', 'photo_path' )
                      ->disk( 's3' )
                      ->storeAs( function ( Request $request ) {
                          return $request->photo_path->getClientOriginalName();
                      } )->rules( [ 'required' ] ),
```
to:
```

 ImageField::make( 'Bild', 'photo_path' )
                      ->disk( 's3' )
                      ->storeOriginalName( 'photo_path' )
                      ->rules( [ 'required' ] ),

```

the right value is set in the database column but the image is not uploaded to s3 anymore?

**Comment by crynobone** [comment, marked as answer]
You are overriding `photo_path` with original name. You should instead set a different attribute to store the original name as suggested in the documentation: 

![image](https://user-images.githubusercontent.com/172966/165315884-cf65200b-20ae-469f-80e4-861e5f98ec0c.png)

**Comment by gepopp** [comment]
Thx allot!
